admin_manual/maintenance/backup.rst

```diff
@@ -7,7 +7,7 @@ When you backup your ownCloud server, there are four things that you need to cop
 #. Your ``config/`` directory.
 #. Your ``data/`` directory.
 #. Your ownCloud database.
-#. Your custom theme files, if you have any. (See `Theming ownCloud <https://doc.owncloud.org/server/9.2/developer_manual/core/theming.html>`_)
+#. Your custom theme files, if you have any. (See `Theming ownCloud <https://doc.owncloud.org/server/10.0/developer_manual/core/theming.html>`_)
```

Comment:
As with the related tickets, I don't agree that this should be part of the backport.